Dockerfile

```diff
@@ -1,22 +1,33 @@
 FROM debian:jessie
 
-ARG DOWNLOAD_URL="https://s3-us-west-2.amazonaws.com/grafana-releases/master/grafana_latest_amd64.deb"
+ARG GRAFANA_VERSION="latest"
+ARG GF_HOME="/usr/share/grafana"
 
-RUN apt-get update && \
-    apt-get -y --no-install-recommends install libfontconfig curl ca-certificates && \
-    apt-get clean && \
-    curl ${DOWNLOAD_URL} > /tmp/grafana.deb && \
-    dpkg -i /tmp/grafana.deb && \
-    rm /tmp/grafana.deb && \
-    curl -L https://github.com/tianon/gosu/releases/download/1.10/gosu-amd64 > /usr/sbin/gosu && \
-    chmod +x /usr/sbin/gosu && \
+RUN apt-get update && apt-get install -qq -y wget tar sqlite libfontconfig curl ca-certificates && \
+    wget -O /tmp/grafana.tar.gz https://s3-us-west-2.amazonaws.com/grafana-releases/release/grafana-$GRAFANA_VERSION.linux-x64.tar.gz && \
```

Comment:
You install both curl and wget? Just use `curl ` here and don't use wget.

Reply:
Agreed.

Reply:
Instead of puting the tar in a file, use 
``` 
mkdir -p $GF_PATHS_HOME && \
wget -qO- $GF_URL/grafana-$GRAFANA_VERSION.linux-x64.tar.gz |tar xfvz - --strip-components=1 --exclude=tools -C $GF_PATHS_HOME
```
Skips writing the file to a file and need to cleanup, as well as directly extracts in the `GF_PATH_HOME`.

Reply:
Nice. We need tools though. May change in the future.